subprojects/docs/src/samples/userguide/dependencyManagement/troubleshooting/cache/dynamic/build.gradle

```diff
@@ -5,7 +5,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'org.springframework:spring-web:5.+'
+    implementation 'org.slf4j:slf4j-api:1.7.+'
```

Comment:
Please check the actual user guide text here. We cannot just change the dependency. The surrounding text refers to the specific dependency and renders a build scan for that concrete example.